e2e/test/iothub/service/ServiceClientE2ETests.cs

```diff
@@ -35,9 +35,8 @@ public async Task Message_NoTimeoutPassed()
 
         private async Task FastTimeout()
         {
-            TimeSpan timeout = TimeSpan.FromTicks(10).Negate();
-            using var cts = new CancellationTokenSource(timeout);
-            await TestTimeout(cts.Token).ConfigureAwait(false);
+            var cancellationToken = new CancellationToken(true);
```

Comment:
creating new CancellationToken(true) for an already cancelled cancellation token.